### PR TITLE
[#4580] feat(catalog): unify cloud storage configurations

### PR DESCRIPTION
**Pull request by FANNG1**
### What changes were proposed in this pull request?
unify the storage configurations

### Why are the changes needed?

Fix: #4580 

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
existing tests

### CODE REVIEW

**Review by diqiu50 — COMMENTED** [review]
Do the related documents need to be updated?

**Comment by FANNG1** [comment]
> Do the related documents need to be updated?

There seems no need to update the document, this is internal,  the other catalog like fileset or engine could reuse the storage properties.

File: catalogs/catalog-common/src/main/java/org/apache/gravitino/storage/OSSProperties.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// Defines the unified OSS properties for different catalogs and connectors.
+public class OSSProperties {
+
+  // The endpoint of Aliyun OSS service.
+  public static final String GRAVITINO_OSS_ENDPOINT = "oss-endpoint";
+  // The static access key ID used to access OSS data.
+  public static final String GRAVITINO_OSS_ACCESS_KEY_ID = "oss-access-key-id";
+  // The static secret access key used to access OSS data.
+  public static final String GRAVITINO_OSS_ACCESS_KEY_SECRET = "oss-access-key-secret";
+
```
Comment (diqiu50) [inline catalogs/catalog-common/src/main/java/org/apache/gravitino/storage/OSSProperties.java:30]: The `oss-access-key-secret` is mismatch with the docs. is it right?

File: catalogs/catalog-common/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergConstants.java [hunk 2]
```diff
@@ -37,20 +37,13 @@ public class IcebergConstants {
...
   public static final String ICEBERG_S3_SECRET_ACCESS_KEY = "s3.secret-access-key";
-  public static final String GRAVITINO_S3_REGION = "s3-region";
   public static final String AWS_S3_REGION = "client.region";
 
-  public static final String GRAVITINO_OSS_ENDPOINT = "oss-endpoint";
   public static final String ICEBERG_OSS_ENDPOINT = "oss.endpoint";
-  public static final String GRAVITINO_OSS_ACCESS_KEY_ID = "oss-access-key-id";
   public static final String ICEBERG_OSS_ACCESS_KEY_ID = "client.access-key-id";
-  public static final String GRAVITINO_OSS_ACCESS_KEY_SECRET = "oss-access-key-secret";
   public static final String ICEBERG_OSS_ACCESS_KEY_SECRET = "client.access-key-secret";
```
Comment (jerryshao) [inline catalogs/catalog-common/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergConstants.java:47]: Do we have the GCS related configurations?

File: catalogs/catalog-common/src/main/java/org/apache/gravitino/storage/OSSProperties.java (hunk 1 above)
Comment (FANNG1) [reply catalogs/catalog-common/src/main/java/org/apache/gravitino/storage/OSSProperties.java:30]: updated the docs to use access key secret for OSS

**Review by diqiu50 — APPROVED** [review]
LGTM

**Pull request merged**